addons/logistics_wirecutter/functions/fnc_interactEH.sqf

```diff
@@ -1,75 +1,80 @@
                 params ["", "_player", "_attachedFence"];
+
                 [_player, _attachedFence] call FUNC(cutDownFence);
             };
             private _fncCondition = {
                 params ["_helper", "_player", "_attachedFence"];
-                if (!([_player, _attachedFence, ["isNotSwimming"]] call EFUNC(common,canInteractWith))) exitWith {false};
-                ((!isNull _attachedFence) && {(damage _attachedFence) < 1} && {("ACE_wirecutter" in (items _player))} && {
-                    //Custom LOS check for fence
-                    private _headPos = ACE_player modelToWorldVisual (ACE_player selectionPosition "pilot");
-                    ((!(lineIntersects [AGLtoASL _headPos, AGLtoASL (_helper modelToWorldVisual [0,0,1.25]), _attachedFence, ACE_player])) ||
-                    {!(lineIntersects [AGLtoASL _headPos, getPosASL _attachedFence, _attachedFence, ACE_player])})
-                })
-            };
 
+                !isNull _attachedFence
+                && {damage _attachedFence < 1}
+                && {"ACE_wirecutter" in ([_player, false, true, true, true, false] call CBA_fnc_uniqueUnitItems)}
+                && {[_player, _attachedFence, ["isNotSwimming"]] call EFUNC(common,canInteractWith)}
+                && {
+                    // Custom LOS check for fence
+                    private _headPos = AGLtoASL (_player modelToWorldVisual (_player selectionPosition "pilot"));
+                    !lineIntersects [_headPos, AGLtoASL (_helper modelToWorldVisual [0, 0, 1.25]), _attachedFence, _player]
+                    || {!lineIntersects [_headPos, getPosASL _attachedFence, _attachedFence, _player]}
+                }
+            };
             {
-                if (!(_x in _fencesHelped)) then {
-                    if ([_x] call FUNC(isFence)) then {
-                        _fencesHelped pushBack _x;
-                        private _helper = "ACE_LogicDummy" createVehicleLocal (getpos _x);
-                        private _action = [QGVAR(helperCutFence), (localize LSTRING(CutFence)), QPATHTOF(ui\wirecutter_ca.paa), _fncStatement, _fncCondition, {}, _x, {[0,0,0]}, 5.5, [false, false, false, false, true]] call EFUNC(interact_menu,createAction);
-                        [_helper, 0, [],_action] call EFUNC(interact_menu,addActionToObject);
-                        _helper setPosASL ((getPosASL _x) vectorAdd [0,0,1.25]);
-                        _addedHelpers pushBack _helper;
-                    };
+                if (!(_x in _fencesHelped) && {_x call FUNC(isFence)}) then {
+                    _fencesHelped pushBack _x;
+                    private _helper = "ACE_LogicDummy" createVehicleLocal [0, 0, 0];
+                    private _action = [QGVAR(helperCutFence), localize LSTRING(CutFence), QPATHTOF(ui\wirecutter_ca.paa), _fncStatement, _fncCondition, {}, _x, {[0, 0, 0]}, 5.5, [false, false, false, false, true]] call EFUNC(interact_menu,createAction);
```

Comment:
What about creating a helper object with this action in config? The attached fence would then be a variable on the fence.